src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java

```diff
@@ -1,20 +1,38 @@
 package uk.gov.hmcts.reform.sscs.reference.data.mappings;
 
+import java.util.Arrays;
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
 @Getter
 @RequiredArgsConstructor
 public enum CancellationReason {
 
-    WITHDRAWN("BBA3-withdrawn", "Withdrawn"),
-    STRUCK_OUT("BBA3-struckOut", "Struck Out"),
-    PARTY_UNABLE_TO_ATTEND("BBA3-partyUnableToAttend", "Party unable to attend"),
     EXCLUSION("BBA3-exclusion", "Exclusion"),
+    INCOMPLETE_TRIBUNAL("BBA3-incompleteTribunal", "Incomplete Tribunal"),
     LAPSED("BBA3-lapsed", "Lapsed"),
-    OTHER("BBA3-other", "Other");
+    LISTED_IN_ERROR("BBA3-listedInError", "Listed In Error"),
+    OTHER("BBA3-other", "Other"),
+    PARTY_DID_NOT_ATTEND("BBA3-partyDidNotAttend", "Party Did Not Attend"),
+    PARTY_UNABLE_TO_ATTEND("BBA3-partyUnableToAttend", "Party unable to attend"),
+    STRUCK_OUT("BBA3-struckOut", "Struck Out"),
+    WITHDRAWN("BBA3-withdrawn", "Withdrawn");
 
     private final String hmcReference;
-    private final String value;
+    private final String label;
+
+    public static CancellationReason getCancellationReasonByHmcReference(String value) {
+        return Arrays.stream(CancellationReason.values())
+                .filter(sl -> sl.getHmcReference().equals(value))
```

Comment:
optional: You know already I don't like this lol.. what is `sl`? maybe change this to cancellationReason? and one below too.

Reply:
Will do